chia/simulator/start_simulator.py

```diff
@@ -22,54 +24,102 @@
 
 
 def service_kwargs_for_full_node_simulator(root_path: Path, config: Dict, bt: BlockTools) -> Dict:
-    path_from_root(root_path, config["database_path"]).parent.mkdir(parents=True, exist_ok=True)
+    path_from_root(root_path, config[SERVICE_NAME]["database_path"]).parent.mkdir(parents=True, exist_ok=True)
     constants = bt.constants
 
     node = FullNode(
-        config,
+        config=config[SERVICE_NAME],
         root_path=root_path,
         consensus_constants=constants,
         name=SERVICE_NAME,
     )
 
-    peer_api = FullNodeSimulator(node, bt)
-    network_id = config["selected_network"]
+    peer_api = FullNodeSimulator(node, bt, config)
+    network_id = config[SERVICE_NAME]["selected_network"]
     kwargs = dict(
         root_path=root_path,
         node=node,
         peer_api=peer_api,
         node_type=NodeType.FULL_NODE,
-        advertised_port=config["port"],
+        advertised_port=config[SERVICE_NAME]["port"],
         service_name=SERVICE_NAME,
-        server_listen_ports=[config["port"]],
+        server_listen_ports=[config[SERVICE_NAME]["port"]],
         on_connect_callback=node.on_connect,
-        rpc_info=(SimulatorFullNodeRpcApi, config["rpc_port"]),
+        rpc_info=(SimulatorFullNodeRpcApi, config[SERVICE_NAME]["rpc_port"]),
         network_id=network_id,
     )
     return kwargs
 
 
+def create_block_tools_simulator(
```

Comment:
Having a new function here just adds a bunch of lines that we don't need. We can just create the BlockTools directly within main